net/munge/Portfile

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
```

Comment:
I've just done the squash, which involves the following steps:

1a) Do a `git rebase -S -i HEAD~X`, where X is _one more_ than the number of revisions I plan on squashing.  So, if I want to squash three commits into one, I'd run `git rebase -i HEAD~3`.
1b) In the text editor that appears, change _all but the first_ instance of `pick` to `s` (or `squash`).  This tells Git that I want to leave the first commit alone, and that I want to squash all previous commits into the first one.  Save and exit the editor.
1c) A new text editor will appear, showing a new (combined) commit message.  Edit as appropriate, and save.
1d) Upon exiting the editor, a GPG prompt will appear, for me to sign my commit.
2) Run `git push --force`.  That does a force-push to Github, causing the new (squashed) commit to appear here!